docs/software/gaming.md

```diff
@@ -359,10 +360,19 @@ Website: <http://www.dxx-rebirth.com/>
 
 ![Steam on Linux logo](../assets/images/dietpi-software-games-steam.jpg){: width="500px"}
 
-### Installation notes
+=== "Installation notes"
 
-- Steam is installed to the DietPi user data directory `/mnt/dietpi_userdata/steam`
-- By default, installing games will install them to `/mnt/dietpi_userdata/steam`
+    - Steam is installed to the DietPi user data directory `/mnt/dietpi_userdata/steam`
+    - By default, installing games will install them to `/mnt/dietpi_userdata/steam`
```

Comment:
The symlink to `/mnt/dietpi_userdata/steam` is btw only created for the root user, while usually games should not be launched as root. Every other user will have Steam installed into it's home directory. A dedicated Steam service user would be great, but too much for this DietPi version.

Not sure if we shall add the note that the above is for root only, or simply remove the note, as `/root/.steam` still exists? We could even remove the symlink from our install code to avoid confusion, until we found a good way to have it for all users in a future DietPi version?

Reply:
I think that we should just temporarily remove the note.

Reply:
Agreed. The symlink is transparent (and transparently created) anyway, so no much benefit to have this as an extra note.